lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveAdmin
+  class ErrorHandler
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      @app.call(env)
+    rescue Exception => e
+      if ActiveAdmin.application.default_namespace.to_s == env['PATH_INFO'].split('/').second
+        render_exception(env, e)
+      else
+        @app.call(env)
+      end
+    end
```

Comment:
I have tried it and it has worked. The controller name depends on the active admin namespace name. Maybe you need to check your test setup. Take a look on `rake routes`